### DISCUSSION TITLE
Any tutorial or example prpject

### DISCUSSION THREAD

**Discussion by Chatelo** [samyakjain101/django-phone-auth#18] [General]
I have setup the library well and even tested with templates for registration, login etc, but now i want to use it with react/nextjs. The issue is that I dont seem to get it with docs. Is there any example project or tutorial I can refer to?

**Comment by samyakjain101** [comment]
This library is purely for django. You will have to develop REST APIs yourself and then use those APIs on react/nextjs to move forward.
You can reference code in this library and replicate.

Here is a tutorial which may point you to right direction:
https://www.youtube.com/watch?v=xjMP0hspNLE
